packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionScreen.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { Dispatch, SetStateAction, useEffect, useMemo } from 'react';
+import { useAvailableMT5Accounts, useMT5AccountsList } from '@deriv/api';
+import { Provider } from '@deriv/library';
+import { qtMerge } from '@deriv/quill-design';
+import { THooks } from '../../../../types';
+import { useDynamicLeverageModalState } from '../../components/DynamicLeverageContext';
+import { Jurisdiction } from '../../constants';
+import { JurisdictionCard } from './JurisdictionCard';
+import { JurisdictionTncSection } from './JurisdictionTncSection';
+
+type TJurisdictionScreenProps = {
+    isCheckBoxChecked: boolean;
+    selectedJurisdiction: THooks.AvailableMT5Accounts['shortcode'];
+    setIsCheckBoxChecked: Dispatch<SetStateAction<boolean>>;
+    setSelectedJurisdiction: Dispatch<SetStateAction<string>>;
+};
+
+const JurisdictionScreen = ({
+    isCheckBoxChecked,
+    selectedJurisdiction,
+    setIsCheckBoxChecked,
+    setSelectedJurisdiction,
+}: TJurisdictionScreenProps) => {
+    const { getCFDState } = Provider.useCFDContext();
+    const { data: availableMT5Accounts } = useAvailableMT5Accounts();
+    const { data: mt5AccountsList } = useMT5AccountsList();
+    const marketType = getCFDState('marketType');
+    const { isDynamicLeverageVisible } = useDynamicLeverageModalState();
+    const jurisdictions = useMemo(
+        () =>
+            availableMT5Accounts
+                ?.filter(account => account.market_type === marketType)
+                .map(account => account.shortcode) || [],
+        [availableMT5Accounts, marketType]
+    );
+    const addedJurisdictions = useMemo(
+        () =>
+            mt5AccountsList
+                ?.filter(account => account.market_type === marketType)
+                .map(account => account.landing_company_short) || [],
+        [marketType, mt5AccountsList]
+    );
+
+    useEffect(() => {
+        setIsCheckBoxChecked(false);
+    }, [selectedJurisdiction, setIsCheckBoxChecked]);
+
+    return (
+        <div
+            className={qtMerge(
+                'flex flex-col h-auto w-[85vw] items-center justify-center my-auto mx-1500 sm:h-[75vh] transition-all ease-in duration-[0.6s]',
+                isDynamicLeverageVisible && '[transform:rotateY(-180deg)] h-[700px] opacity-50'
```

Comment:
Can try this
```suggestion
                isDynamicLeverageVisible && 'transform -rotate-180 translate-y-50 h-[700px] opacity-50'
```

Reply:
I don't think this will work, Instead of flipping it will rotate the screen turning it upside down. rotateY comes under 3d transformation and tailwind doesn't support 3d transformations, that's why it was added like this.

Reply:
Okay thanks for checking it out